sample-client-app/src/main/resources/sampleconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "product": "JavaSample",
+  "version": "1.0.0",
```

Comment:
'product' and 'version' should be removed? since they are defined in 'online_service_url'.

Reply:
I will remove these 2 properties once the online_service_url has the correct value. I am waiting because there is an ongoing effort to improve API endpoint URLS in the service, right?